tests/scripts/generate_test_keys.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+"""Module generating EC and RSA keys to be used in test_suite_pk instead of
+generating the required key at run time. This helps speeding up testing."""
+
+import os
+import sys
+# pylint: disable=wrong-import-position
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__)) + "/"
+sys.path.append(SCRIPT_DIR + "../../scripts/")
```

Comment:
```suggestion
import scripts_path # pylint: disable=unused-import
```

This should take care of the mypy path problem

Reply:
Thanks a lot  for the suggestion!!!
However I implemented it in #9017 since this is where the [mypy lint issue](https://github.com/Mbed-TLS/mbedtls/pull/9017#discussion_r1560624698) appeared.

Reply:
Seems to me the problematic `export MYPYPATH="../../scripts"` is in this PR too.

Reply:
Right. I thought that it only affected #9010 because the CI was fine here, but I was wrong. This should be fixed now.